### PR TITLE
8/Server - Listen

### PR DESCRIPTION
**Pull request by LucasTesnier**
Add a custom value for the listen queue.

Change the max clientNb value.

### CODE REVIEW

File: src/zappy_server_src/core/argument_handling.c [hunk 1]
```diff
@@ -89,7 +89,7 @@ static bool process_integrity(argument_t *args)
         dprintf(2, "Please give a valid map size.\n");
         return false;
     }
-    if (args->client_nb == 0) {
+    if (args->client_nb == 0 || args->client_nb > 8) {
```
Comment (31Nathan) [inline src/zappy_server_src/core/argument_handling.c:92]: Why 8 ?

**Review by 31Nathan — APPROVED** [review]
LGTM! Just have one comment

**Review by Cavonstavant — APPROVED** [review]

File: src/zappy_server_src/core/argument_handling.c (hunk 1 above)
Comment (LucasTesnier) [reply src/zappy_server_src/core/argument_handling.c]: Fix

**Review by osvegn — APPROVED** [review]
Look Good To Me

**Pull request merged**